vendor/github.com/OpenBazaar/multiwallet/bitcoin/wallet.go

```diff
@@ -213,6 +214,31 @@ func (w *BitcoinWallet) Transactions() ([]wi.Txn, error) {
 
 func (w *BitcoinWallet) GetTransaction(txid chainhash.Hash) (wi.Txn, error) {
 	txn, err := w.db.Txns().Get(txid)
+	if err == nil {
+		tx := wire.NewMsgTx(1)
+		rbuf := bytes.NewReader(txn.Bytes)
+		err := tx.BtcDecode(rbuf, wire.ProtocolVersion, wire.WitnessEncoding)
+		if err != nil {
+			return txn, err
+		}
+		outs := []wi.TransactionOutput{}
+		for i, out := range tx.TxOut {
+			_, addrs, _, err := txscript.ExtractPkScriptAddrs(out.PkScript, w.params)
+			if err != nil {
+				return txn, err
+			}
+			if len(addrs) == 0 {
```

Comment:
This code in each of the wallet implementations has a small intentional of putting us in an unrecoverable state. For example, suppose sender of this transaction included a change output that was _not_ of a standard address type. The change output is for their wallet, not ours, so technically this would still be a valid transaction on the network. However, our code would error here. Either where `err != nil` or where `len(addrs) == 0`.

So I think it's probably best to just set the address to "" in this case rather than error. Maybe log a warning as well.